src/services/expense.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+class Expense {
```

Comment:
From `Expense` name I expect that I this is an object of Expense interface or it's class/constructor